### PR TITLE
[Docs] Add header for files

### PR DESCRIPTION
**Pull request by xvjiarui**

### CODE REVIEW

**Comment by innerlee** [comment]
May you put the script in `tools/`

File: mmcv/cnn/bricks/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# Copyright (c) Open-MMLab. All rights reserved.
```
Comment (hellock) [inline mmcv/cnn/bricks/__init__.py:1]: We need to change all `Open-MMLab` to `OpenMMLab`.

**Comment by hellock** [comment]
Apply this to cpp/cu/cuh files too.

**Comment by xvjiarui** [comment]
@hellock done

**Comment by hellock** [comment]
In some files, e.g., cu, cpp, Open-MMLab is not replaced with OpenMMLab.

**Comment by xvjiarui** [comment]
@hellock All files are replaced.

**Review by hellock — APPROVED** [review]

**Pull request merged**